assets/components/directDebit/directDebitActions.js

```diff
@@ -56,15 +58,19 @@ const setDirectDebitFormError = (message: string): Action =>
 const resetDirectDebitFormError = (): Action =>
   ({ type: 'DIRECT_DEBIT_RESET_FORM_ERROR' });
 
+const transitionConfirmationView = (): Action =>
```

Comment:
This two actions could be replaced with just one action which receives the new phase as an argument.
``
const setProcessPhase = (phase: Phase): Action => 
({type: SET_PROCESS_PHASE, phase })
```